test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestJstack.java

```diff
@@ -44,72 +44,12 @@
 import org.testng.annotations.BeforeSuite;
 import org.testng.annotations.Test;
 
-import openj9.tools.attach.diagnostics.info.JvmThreadInfo;
-import openj9.tools.attach.diagnostics.info.ThreadGroupInfo;
-
 @Test(groups = { "level.extended" })
 public class TestJstack extends AttachApiTest {
 
 	private static final String JSTACK_COMMAND = "jstack"; //$NON-NLS-1$
 	Object syncObject = new Object();
 
-	@Test
-	public void testSingleThread() throws IOException {
```

Comment:
Rather than delete these tests, please rework them for the new organization (the functionality remains, so should the tests).

Reply:
The previous implementation of `jstack` used a number of new classes; this file contained  unit tests for these internal classes.  As the classes are deleted, I deleted the corresponding unit tests but retained the tests for `jstack` per se.  I will update the commit to clarify.

Reply:
Where are the equivalent tests for `ThreadInfo` or `ThreadInfoBase`?

Reply:
`ThreadInfo` is tested via the existing test `test/functional/JLM_Tests/src_80/org/openj9/test/java/lang/management/TestThreadInfo.java`. There are no new tests for `ThreadInfoBase`, since it is basically a container and all its methods are mirrors of `ThreadInfo`'s.